web/src/containers/admin/MyAccount.js

```diff
@@ -74,11 +74,11 @@ class MyAccount extends Component {
             </Link>
           </div>
         </header>
-
-        <div className="ds-l-container card">
-          <div className="ds-l-row ds-u-radius ds-u-padding-y--5">
-            <div className="ds-l-col--1 ds-u-fill--white ds-u-margin-left--auto" />
-            <div className="ds-l-col--6 ds-u-fill--white ds-u-padding-y--5">
```

Comment:
I made an executive decision (ha) here: I only left in the classes that governed structure. I moved rules that govern style (border radius, background color) into the CSS.